applications/nrf5340_audio/src/modules/music_color_sync.h

```diff
@@ -0,0 +1,22 @@
+
+#ifndef _MUSIC_COLOR_SYNC_H_
+#define _MUSIC_COLOR_SYNC_H_
+
+#include <zephyr/kernel.h>
+
+/**@brief Updates decoded pcm data used for FFT 
+ * 
+ * @param[in] pcm_data_mono pointer to decoded pcm data (mono)
+ * @param[in] len Size of decoded data
+ * 
```

Comment:
Should have an explanation here of how the data is used inside the module

Reply:
Added small explanation